text/0000-target-tier-policy.md

```diff
@@ -0,0 +1,623 @@
+(and in particular how they can make it available via `rustup`), and
+occasionally ask what it would take to add a new tier 1 target. The Rust
+project has no clear policy for target tiers. People not only don't know, they
+don't know who to ask or where to start.
+
+See <https://forge.rust-lang.org/release/platform-support.html> for more
+information about targets and tiers.
+
+Once accepted, the policy sections of this RFC should be posted on
+<https://forge.rust-lang.org/> in a "Target Tier Policy" section; this RFC will
+not be the canonical home of the up-to-date target tier policy.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+Rust provides three tiers of target support:
+
+- Rust provides no guarantees about tier 3 targets.
+- Rust's continuous integration checks that tier 2 targets will always build,
+  but they may not pass tests.
+- Rust's continuous integration checks that tier 1 targets will always build
+  and pass tests.
+
+Adding a new tier 3 target imposes minimal requirements; we focus primarily on
+avoiding disruption to other ongoing Rust development.
+
+Tier 2 and tier 1 targets place work on the Rust community as a whole, to avoid
+breaking the target. Thus, these tiers require commensurate and ongoing efforts
+from the maintainers of the target, to demonstrate value and to minimize any
+disruptions to ongoing Rust development.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+Rust provides three tiers of target support:
+
+- Rust provides no guarantees about tier 3 targets.
+- Rust's continuous integration checks that tier 2 targets will always build,
+  but they may or may not pass tests.
+- Rust's continuous integration checks that tier 1 targets will always build
+  and pass tests.
+
+This policy defines the requirements for accepting a proposed target at a given
+level of support.
+
+Each tier builds on all the requirements from the previous tier, unless
+overridden by a stronger requirement. Tier 2 and tier 1 also provide additional
+requirements to be met if supplying host tools for the target; a target at tier
+2 or tier 1 is not required to supply host tools, but if it does, it must meet
+the corresponding additional requirements for host tools.
+
+The policy for each tier also documents the Rust governance teams that must
+approve the addition of any target at that tier. Those teams are responsible
+for reviewing and evaluating the target, based on these requirements and their
+own judgment. Those teams may apply additional requirements, including
+subjective requirements, such as to deal with issues not foreseen by this
+policy. (Such requirements may subsequently motivate additions to this policy.)
+
+While these criteria attempt to document the policy, that policy still involves
+human judgment. Targets must fulfill the spirit of the requirements as well, as
```

Comment:
In this case, I mean the spirit of each individual requirement, rather than the letter. I don't think an additional summary of the ethos of each target helps in this particular case.